src/components/drag-and-drop/draggable-context/draggable-context.js

```diff
@@ -78,13 +89,27 @@ class DraggableContext extends React.Component {
   }
 
   static defaultProps = {
-    customDragLayer: <CustomDragLayer />
+    customDragLayer: <CustomDragLayer />,
+    autoScroll: false
   }
 
   state = {
     activeIndex: null // {Number} tracks the currently dragged index
   }
 
+  constructor(props) {
+    super(props);
+
+    // Default speed of auto scrolling
+    this.speed = 0;
+
+    // Frame callback ref
+    this.frame = null;
+
+    // Frame callback ref
+    this.element = null;
```

Comment:
does this need to be an instance variable? it seems to be only referenced in the `tick` method